src/components/input-time/input-time.e2e.ts

```diff
@@ -0,0 +1,360 @@
+import { newE2EPage } from "@stencil/core/testing";
+import { localizeTimeString, localizeTimeStringToParts } from "../../utils/time";
+import {
+  accessible,
+  defaults,
+  disabled,
+  focusable,
+  formAssociated,
+  labelable,
+  reflects,
+  renders
+} from "../../tests/commonTests";
+import { CSS } from "./resources";
+
+describe("calcite-input-time", () => {
+  it("renders", async () => renders("calcite-input-time", { display: "inline-block" }));
+
+  it("is accessible", async () =>
+    accessible(`
+    <calcite-label>
+      Input Time
+      <calcite-input-time name="test"></calcite-input-time>
+    </calcite-label>
+  `));
+
+  it("has defaults", async () =>
+    defaults("calcite-input-time", [
+      { propertyName: "scale", defaultValue: "m" },
+      { propertyName: "step", defaultValue: 60 }
+    ]));
+
+  it("reflects", async () =>
+    reflects(`calcite-input-time`, [
+      { propertyName: "disabled", value: true },
+      { propertyName: "iconFlipRtl", value: true },
+      { propertyName: "required", value: true },
+      { propertyName: "scale", value: "m" }
+    ]));
+
+  it("is labelable", async () => labelable("calcite-input-time"));
+
+  it("should focus the hour input when setFocus is called", async () =>
+    focusable(`calcite-input-time`, {
+      shadowFocusTargetSelector: "span.hour"
+    }));
+
+  it("can be disabled", () => disabled("calcite-input-time"));
+
+  it("when set to readOnly, element still focusable but won't display the controls or allow for changing the value", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-input-time id="canReadOnly" readonly></calcite-input-time>`);
+
+    const component = await page.find("calcite-input-time");
+
+    await component.callMethod("setFocus");
+    await page.waitForChanges();
+
+    expect(await page.evaluate(() => document.activeElement.id)).toBe("canReadOnly");
```

Comment:
this `expect` is unnecessary and it's validating anything from the component so it should be removed